activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -373,6 +393,19 @@ def group!(*args) # :nodoc:
     #
     #   User.order('name DESC, email')
     #   # SELECT "users".* FROM "users" ORDER BY name DESC, email
+    #
+    # === Arel
+    #
+    # If you need to pass in complicated expressions that you have verified
+    # are safe for the DB, you can use Arel.
+    #
+    #   User.order(Arel.sql('end_date - start_date'))
+    #   # SELECT "users".* FROM "users" ORDER BY end_date - start_date
+    #
+    # Custom DB syntax like JSON columns for Postgres are supported in this way.
```

Comment:
Last comment from me 🙂 🙏 
We don't use the abbreviation `DB` in this file (except twice in this PR).

Would it be clearer to use `database` (which is added on line 381) and add two commas?
```suggestion
    # Custom database syntax, like JSON columns for Postgres, is supported in this way.
```

Or maybe even:
```suggestion
    # Custom query syntax, like JSON columns for Postgres, is supported in this way.
```

Reply:
Committed.